setup.py

```diff
@@ -540,7 +540,7 @@ def setup_package():
     #            in N+1 will turn into errors in N+3
     # For Python versions, if releases is (e.g.) <=3.9.x, set bound to 3.10
     np_minversion = '1.17.3'
-    np_maxversion = '9.9.99'
+    np_maxversion = '1.24.0'
     python_minversion = '3.8'
     python_maxversion = '3.10'
```

Comment:
Ooh, this `python_maxversion` may need a change too based on CI, probably isn't getting flushed in the `master` CI